objects/rct2ww/ride/rct2.ww.tigrtwst.json

```diff
@@ -100,6 +100,7 @@
             "cs-CZ": "Horská dráha, která se točí dokola tak, že návštěvníci jsou pořád na stejném místě",
             "ko-KR": "하트라인 트위스트에서 사용할 수 있는 롤러코스터 차량",
             "pl-PL": "Kolejka górska wykonująca zabójcze zakręty",
+            "pt-BR": "Montanhas russas com inversões de 360 graus",
```

Comment:
Again, this appears to be a translation from French. If Portuguese has no proper translation for "heartline twist", change "360-degree inversions" to "360-degree **lateral** inversions".